scripts/variantstore/wdl/GvsCreateVATFromAnnotations.wdl

```diff
@@ -0,0 +1,549 @@
+        String table_suffix
+        String? service_account_json_path
+        Array[String] prep_jsons_done
+    }
+
+    # There are two pre-vat tables. A variant table and a genes table. They are joined together for the vat table
+
+    String vat_table = "vat_" + table_suffix
+    String variant_transcript_table = "vat_vt_"  + table_suffix
+    String genes_table = "vat_genes_" + table_suffix
+
+    String vt_path = output_path + 'vt/*'
+    String genes_path = output_path + 'genes/*'
+
+    String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+
+    command <<<
+
+       echo "project_id = ~{project_id}" > ~/.bigqueryrc
+
+       DATE=86400 ## 24 hours in seconds
+
+
+       if [ ~{has_service_account_file} = 'true' ]; then
+            gsutil cp ~{service_account_json_path} local.service_account.json
+            export GOOGLE_APPLICATION_CREDENTIALS=local.service_account.json
+            gcloud auth activate-service-account --key-file=local.service_account.json
+            gcloud config set project ~{project_id}
+       fi
+
+       set +e
+       bq show --project_id ~{project_id} ~{dataset_name}.~{variant_transcript_table} > /dev/null
+       BQ_SHOW_RC=$?
+       set -e
+
+       if [ $BQ_SHOW_RC -ne 0 ]; then
+         echo "Creating a pre-vat table ~{dataset_name}.~{variant_transcript_table}"
+         bq --location=US mk --expiration=$DATE --project_id=~{project_id}  ~{dataset_name}.~{variant_transcript_table} ~{vt_schema}
+       fi
+
+       echo "Loading data into a pre-vat table ~{dataset_name}.~{variant_transcript_table}"
+       echo ~{vt_path}
+       echo ~{genes_path}
+       bq --location=US load --project_id=~{project_id} --source_format=NEWLINE_DELIMITED_JSON ~{dataset_name}.~{variant_transcript_table} ~{vt_path}
+
+       set +e
+       bq show --project_id ~{project_id} ~{dataset_name}.~{genes_table} > /dev/null
+       BQ_SHOW_RC=$?
+       set -e
+
+       if [ $BQ_SHOW_RC -ne 0 ]; then
+         echo "Creating a pre-vat table ~{dataset_name}.~{genes_table}"
+         bq --location=US mk --expiration=$DATE --project_id=~{project_id}  ~{dataset_name}.~{genes_table} ~{genes_schema}
+       fi
+
+       echo "Loading data into a pre-vat table ~{dataset_name}.~{genes_table}"
+       bq --location=US load  --project_id=~{project_id} --source_format=NEWLINE_DELIMITED_JSON  ~{dataset_name}.~{genes_table} ~{genes_path}
+
+       set +e
+       bq show --project_id ~{project_id} ~{dataset_name}.~{vat_table} > /dev/null
+       BQ_SHOW_RC=$?
+       set -e
+
+       if [ $BQ_SHOW_RC -ne 0 ]; then
+         echo "Creating the vat table ~{dataset_name}.~{vat_table}"
+         bq --location=US mk --project_id=~{project_id} ~{dataset_name}.~{vat_table} ~{nirvana_schema}
+       else
+         bq rm -t -f --project_id=~{project_id} ~{dataset_name}.~{vat_table}
+         bq --location=US mk --project_id=~{project_id} ~{dataset_name}.~{vat_table} ~{nirvana_schema}
```

Comment:
The `mk` line could be outside the `if`/`else` to avoid duplication

Reply:
yes--I think this whole gate should be flipped actually